services/grafana-loki/0.33.2/jobs.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: grafana-loki-jobs
+  namespace: ${releaseNamespace}
+spec:
+  force: true  # Recreate jobs when mutations are disallowed.
+  prune: true
+  wait: true
+  interval: 6h
+  retryInterval: 1m
+  timeout: 1m
+  path: ./services/grafana-loki/0.33.2/jobs
+  sourceRef:
+    kind: GitRepository
+    name: management
+    namespace: kommander-flux
+  postBuild:
+    substituteFrom:
+      - kind: ConfigMap
+        name: substitution-vars
```

```suggestion
  # passing releaseNamespace to 2nd level configuration files for ability to configure namespace correctly in attached clusters
  # Using `substituteFrom` with `substitution-vars` creates 2nd level resources in `kommander` namespace instead of workspace ns
  postBuild:
    substitute:
      releaseNamespace: ${releaseNamespace}
```
I think we need this (`substitution-vars` only exists on mgmt cluster)